gslb/cache/controller_obj_cache.go

```diff
@@ -687,7 +693,7 @@ func GetDetailsFromAviGSLBFormatted(gsObj models.GslbService) (uint32, []GSMembe
 			if server == "" {
 				server = ipAddr
 			}
-			serverList = append(serverList, server+"-"+strconv.Itoa(int(weight)))
+			serverList = append(serverList, server+"-"+strconv.Itoa(int(weight))+"-"+strconv.Itoa(int(priority)))
```

Comment:
if priority is not set, it would append a 0 in the end? Do you want the priority now to be associated with every server even if it's not requested?

Reply:
I think it will be 10? Per the code logic below.

Reply:
Priority is always set. For the earlier versions, it was 10. With this PR, the default is 10, unless otherwise changed.